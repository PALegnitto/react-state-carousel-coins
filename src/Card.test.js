import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";
import TEST_IMAGES from "./_testCommon.js";

it("renders without crashing", function () {
  // this is a low-value test, but better than nothing
  render(<Card caption="test" currNum="1" src="/static/media/image1.08e1519e765278b10548.jpg" totalNum="1" />);
});

it("matches snapshot", function () {
  const { container } = render(<Card caption="test" currNum="1" src="/static/media/image1.08e1519e765278b10548.jpg" totalNum="1" />);
  expect(container).toMatchSnapshot();
});