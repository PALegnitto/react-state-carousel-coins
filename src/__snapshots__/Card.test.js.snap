// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test
    </h4>
    <img
      alt="test"
      class="Card-image"
      src="/static/media/image1.08e1519e765278b10548.jpg"
    />
    <small
      class="Card-small"
    >
      Image 
      1
       of 
      1
      .
    </small>
  </div>
</div>
`;
